### ISSUE TITLE
Calendar options

### ISSUE THREAD

**Issue by marciz** [Swechhya/excelR#59]
**Describe the bug**
Sub-lists are not working in excelTable columns option. Couldn't get working example with calendar options

**To Reproduce**
```
dt <- iris[1:10, ]['Species']
dt$date <- Sys.Date()
cols <- data.frame(title = colnames(dt), width = 100, options = I(list(NA, list(
  format = 'DD.MM.YYYY', 
  placeholder = 'my date', 
  months = c('1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12')
))))
excelTable(dt, columns = cols)
```

**Comment by Swechhya** [comment]
@marciz  Can you please explain what do you mean by sub-lists? I am not able to understand what you are trying to achieve with the piece of code.

**Comment by marciz** [comment]
I was trying to adjust calendar settings with options parameter
https://bossanova.uk/jexcel/v3/examples/date-and-datetime-picker
Not sure it is correct, but I guess options parameter would be type list

**Comment by Swechhya** [comment]
@marciz This issue has been resolved in PR #70. Feel free to reopen this issue if the bug you reported is not resolved.
